packages/rainbowkit-wallets/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@vocdoni/rainbowkit-wallets",
+  "version": "1.0.0",
+  "license": "MIT",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.es.js",
+  "types": "dist/index.d.ts",
+  "exports": {
+    "import": "./dist/index.es.js",
+    "require": "./dist/index.cjs.js"
+  },
+  "type": "module",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "dev": "NODE_ENV=development yarn build --watch",
+    "lint": "tsc",
+    "build": "vite build",
+    "size": "size-limit",
+    "preview": "vite preview",
+    "prepack": "json -f package.json -I -e \"delete this.devDependencies; delete this.dependencies\""
```

Comment:
I'm not sure you should be doing this. Instead, I'd add the dependencies that have to be bundled to the "devDependencies", and would remove only that (leaving the dependencies intact, with dependencies that will not be bundled [aka installed externally]).